docs/docs/dagster-plus/deployment/alerts.md

```diff
@@ -0,0 +1,322 @@
+---
```

Comment:
not sure why this is a brand new file instead of updating the existing alerts doc... investigating

Reply:
ah i see. this is an older version of the docs page. The current version is `docs/docs/dagster-plus/features/alerts/creating-alerts.md` but that docs page references the code snippets generated by `generate_alerts_doc.py`